src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int RANDOM_LIMIT = 100;
+    private ColorSupplier randomColor = new ColorSupplier();
+    private Random random = new Random();
+
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ball.setColor(randomColor.getRandomColor());
+        ball.setNumber(random.nextInt(RANDOM_LIMIT));
+        return ball;
```

Comment:
Let`s create constructor of Ball with color and number parameters without setters it will looks better)

Reply:
off topic, maybe it would be better to replace loop fori by foreach in the main method?